alf/algorithms/rl_algorithm.py

```diff
@@ -341,6 +357,9 @@ def summarize_rollout(self, experience):
         Args:
             experience (Experience): experience collected from ``rollout_step()``.
         """
+        if not alf.summary.should_record_summaries():
```

Comment:
I think this condition is already checked by the summary wrapper?

Reply:
Removed